app/helpers/games_helper.rb

```diff
@@ -1,2 +1,19 @@
 module GamesHelper
+
+  def custom_pluralize(objectArr, word)
```

Comment:
You can use Rails translations here too.
https://guides.rubyonrails.org/i18n.html#pluralization